modules/core/src/webgl/classes/texture.js

```diff
@@ -315,6 +320,18 @@ export default class Texture extends Resource {
       }
     });
 
+    if (data && data.byteLength) {
+      this._trackAllocatedMemory(data.byteLength);
+    } else {
+      let bytesPerTexel = TEXTURE_FORMATS[this.format].bytesPerTexel;
```

Comment:
Maybe helper methods to calculate this from the table, both here and in other file?